src/SettingStore.php

```diff
@@ -141,8 +150,7 @@ public function save()
 	protected function checkLoaded()
```

Comment:
You may aswell remove this method and do check something like when you call `load` it will check to see if it is already loaded and have a function `forceReload` as I assume that is why you're wanting to extract out the `load` method.